Example/EpoxyExample/ViewControllers/MainViewController.swift

```diff
@@ -98,6 +111,11 @@ final class MainViewController: NavigationController {
       viewController = CardStackViewController()
```

Comment:
We can probably remove this CardStackViewController since it'd be recommended to build with with layout groups now! Or we could also just migrate it to layout groups 🎉

Reply:
we will need to add selection handling to groups first, but then yes!